packages/postman-to-openapi/src/convert.ts

```diff
@@ -0,0 +1,180 @@
+import type { OpenAPIV3 } from '@scalar/openapi-types'
+
+import { processAuth } from './helpers/authHelpers'
+import { processExternalDocs } from './helpers/externalDocsHelper'
+import { processItem } from './helpers/itemHelpers'
+import { processLicenseAndContact } from './helpers/licenseContactHelper'
+import { processLogo } from './helpers/logoHelper'
+import { parseServers } from './helpers/serverHelpers'
+import { normalizePath } from './helpers/urlHelpers'
+import type { PostmanCollection } from './types'
+
+/**
+ * Converts a Postman Collection to an OpenAPI 3.0.0 document.
+ * This function processes the collection's information, servers, authentication,
+ * and items to create a corresponding OpenAPI structure.
+ */
+export function convert(
+  postmanCollection: PostmanCollection,
```

Comment:
Can we allow to pass a string here, too? We’d need to `JSON.parse` it then. But I like it when things are easy with the input and strict with the output.

```suggestion
  postmanCollection: PostmanCollection | string,
```